src/main/java/com/idrsolutions/microservice/JPedalServlet.java

```diff
@@ -145,7 +134,8 @@ protected void convert(String uuid,
         }
 
         //Makes the directory for the output file
-        new File(outputDirStr + fileSeparator + fileNameWithoutExt).mkdirs();
+        File output = new File(outputDirStr + fileSeparator + fileNameWithoutExt);
```

Comment:
Could this be final?
```suggestion
        final File output = new File(outputDirStr + fileSeparator + fileNameWithoutExt);
```